contracts/gateway/MessageBus.sol

```diff
@@ -61,10 +61,6 @@ library MessageBus {
     /** Message */
     struct Message {
 
-        /**
-         * intent hash, this can be staking intent hash, redemption intent
-         * hash, gateway linking hash
-         */
```

Comment:
Why removed?

Reply:
Message bus shouldn't know about a different kind of intent hash(Message bus). I feel there shouldn't be any mention of the stake, redemption etc in the message bus.